app/src/main/res/layout/previous_insurer_item_layout.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?attr/selectableItemBackground"
+    android:padding="@dimen/base_margin_double">
+
+    <ImageView
+        android:id="@+id/icon"
+        android:layout_width="28dp"
+        android:layout_height="28dp"
+        android:layout_gravity="center_vertical"
+        android:layout_marginEnd="@dimen/base_margin_double"
+        tools:ignore="ContentDescription"
```

Comment:
Customarily we've been setting `android:contentDescription` to `@null` when dealing with decorative images, ie not important for screen readers. What do you think about that approach? 🙂

Reply:
Sounds good to me :) Fixing